plugins/compliments/plugin/public/comps_char.rb

```diff
@@ -0,0 +1,17 @@
+module AresMUSH
+
+  class Character < Ohm::Model
+    collection :comps, "AresMUSH::Comps"
+    attribute :comps_given, :type => DataType::Integer
+  end
+end
+
+
+class Comps < Ohm::Model
+  include ObjectModel
+
+  index :character
+  reference :character, "AresMUSH::Character"
+  attribute :comp_msg
+  attribute :from
```

Comment:
Oh, good point. Thanks. I'll leave it here (in part, I think its' actually nice to have the old name associated with the compliment), but try to keep that in mind for the future. I've burned myself on this before (not character names, but things that I turned into strings and stored in a model instead of making it read changes live from config), so it's a good reminder.